### PR TITLE
Undo: remove b16-service-name label

### PR DESCRIPTION
**Pull request by anukul**
Fixes https://github.com/storyscript/platform-engine/issues/410

### CODE REVIEW

File: tests/unit/Kubernetes.py [hunk 1]
```diff
@@ -609,6 +609,7 @@ def test_new_ssl_context():
     tag = image.split(':')[-1]
     service_uuid = '85b6735f-e648-439e-bb18-9b4ad130f69f'
     service_tag_uuid = 'e5103d54-8a49-4619-9519-04a892dd6817'
+    b16_service_name = base64.b16encode('alpine'.encode()).decode()
```
Comment (wilzbach) [inline tests/unit/Kubernetes.py]: use the actual value here.

File: asyncy/Kubernetes.py [hunk 2]
```diff
@@ -567,6 +567,8 @@ def get_liveness_probe(cls, app, service: str):
 
         liveness_probe = cls.get_liveness_probe(app, service_name)
 
+        b16_service_name = base64.b16encode(service_name.encode()).decode()
```
Comment (wilzbach) [inline asyncy/Kubernetes.py]: Out of interest: any reason why we can't use full Base64? 
(blows up the result x4)

Comment (judepereira) [reply asyncy/Kubernetes.py]: Because of k8s value limitations.

**Comment by judepereira** [comment]
Closing in favour of solving this the right way: https://github.com/storyscript/logstreamer/pull/6

**Pull request closed without merging**